hal/ia32/cpu.c

```diff
@@ -77,14 +77,28 @@ struct {
 /* context management */
 
 
+static unsigned int hal_cpuGetTssIndex(void)
+{
+	return GDT_FREE_SEL_IDX + 2 * hal_cpuGetID();
+}
+
+
+unsigned int hal_cpuGetTlsIndex(void)
+{
+	return GDT_FREE_SEL_IDX + 2 * hal_cpuGetID() + 1;
+}
+
+
 u32 cpu_getEFLAGS(void)
 {
 	u32 eflags;
 
-	__asm__ volatile(
-		"pushf; "
-		"popl %0; "
+	/* clang-format off */
+	__asm__ volatile (
+		"pushf\n\t"
+		"popl %0\n\t"
```

Comment:
Redundant `\n\t`, it's added by `gcc` already for the last asm line.
```suggestion
		"popl %0"
```